src/deploy/jenkins-uat.js

```diff
@@ -0,0 +1,22 @@
+const fetch = require('node-fetch');
+
+module.exports = () => {
+    const jenkinsUrl = "https://ctv-jenkins-02.truex.com/generic-webhook-trigger/invoke";
+    return fetch(jenkinsUrl, {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json",
+            "token": "TAR-HTML5_UAT",
+        },
+        body: JSON.stringify({
+            "whakapapaBranchName": "develop",
+            "devicePlatforms": "Unknown,Tizen,LG,XboxOne,FireTV,AndroidTV,Vizio,Comcast",
+            "skylineBranchName": "develop",
+            "skylineS3BranchDirName": "develop",
+            "fromBuildJobUrl": process.env.TRAVIS_BUILD_WEB_URL,
+            "sourceRepo": process.env.TRAVIS_REPO_SLUG
+        })
+    }).catch(error => {
+        console.log(error);
```

Comment:
let's put a bit on context so that the error message is clear:
```
console.error('error invoking Jenkins automation at ' + jenkinsUrl + '\n. ' +error);